backend/handlers/createtrippsot.js

```diff
@@ -1,6 +1,7 @@
 const trip = require('../../database/triphelpers.js')
 module.exports = (req, res) => {
-  const data = Object.assign(JSON.parse(req.payload),  req.state.sid)
+  const user_id = {user_id: req.state.sid.user_id};
+  const data = Object.assign(req.payload, user_id);
```

Comment:
Use camelCase for JavaScript variables.
Also you could get away with just doing `const data = Object.assign(req.payload, { user_id: req.state.sid.user_id });`